src/components/Collapsible/types.ts

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+
+
+export interface Props {
+
+    /**
+     * Classes to be passed through to the component.
+     */
+    className?: string,
+
+    /**
+     * Static JSX, controls open/close functionality.
+     */
+    anchor: ReactNode,
```

Comment:
Maybe we should consider renaming this prop something like `anchorContent` or something? `anchor` seems a little bit nebulous imho.